news/models.py

```diff
@@ -177,17 +185,54 @@ def copy_relations(self, oldinstance):
         """
         self.tags.set(oldinstance.tags.all())
 
-    def get_latest_articles(self, limit=3):
+    @property
+    def category_articles(self):
```

Comment:
This name could be more descriptive to reflect what it does - it filters articles by category. So `filtered_by_category` perhaps?